src/Filter/ExceptionTypeFilter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using log4net.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace log4net.Filter
+{
+    public class ExceptionTypeFilter : FilterSkeleton
+    {
+        public ExceptionTypeFilter()
+        {
+            this.AcceptOnMatch = true;
+        }
+
+        /// <summary>
+        /// Set the exception type name to filter
+        /// </summary>
+        public string ExceptionTypeName { get; set; }
```

Comment:
These properties are not supported on ancient frameworks like .net 2.0. By changing the getter and setter to rely on a backing field this contribution will also build against .net 2.0. Further it allows the implementation to resolve the type behind the string and do the expensive reflection only once and not for any future log event.